content/quickstart/nodejs/metrics.md

```diff
@@ -228,9 +229,9 @@ const statusTagKey = "status";
 const errorTagKey = "error";
 
 const latencyView = stats.createView(
-  "demo/latency",
+  "repl_latency_distribution",
```

Comment:
I'd highly discourage "_" and even the aggregation type in the measure, please revert these.